launch/client.py

```diff
@@ -812,6 +812,34 @@ def get_model_bundle(
         ), f"Bundle with name `{bundle_name}` not found"
         return ModelBundle.from_dict(resp["bundles"][0])  # type: ignore
 
+    def clone_model_bundle_with_changes(
+        self,
+        existing_bundle: Union[ModelBundle, str],
+        new_bundle_name: str,
+        app_config: Dict[str, str],
```

Comment:
should this be optional? (i.e. if it's None or something we use the old app_config, otherwise we use the new one)

Reply:
hmmm i think setting it to null makes more sense?